backoffice/views.py

```diff
@@ -1,12 +1,55 @@
 from django.shortcuts import render
 from django.contrib.auth.decorators import login_required
-from api.models import Animal, Entity
+from api.models import *
```

Comment:
Wildcard imports should be avoided. 
https://www.python.org/dev/peps/pep-0008/#imports